app/assets/javascripts/sequence_show.js

```diff
@@ -512,3 +535,33 @@ function init_sequence_show(data) {
         }
     }
 }
+
+function toggle_column(col) {
```

Comment:
- Don't create new functions in the global namespace except for the init functions (and the ones in application.js). The same goes for `toggle_tab` below.
- Use camelCase for JavaScript function names.


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3471) by @bmesuere on Mon Nov 09 2015 at 22:18._